src/napari/layers/labels/_labels_key_bindings.py

```diff
@@ -105,17 +105,27 @@ def swap_selected_and_background_labels(layer: Labels):
 
 
 @register_label_action(
-    trans._('Decrease the currently selected label by one'),
+    trans._('Select the preceding label.'),
 )
 def decrease_label_id(layer: Labels):
-    layer.selected_label -= 1
+    if layer.predefined_labels is None:
+        layer.selected_label -= 1
+    else:
+        labels = sorted(layer.predefined_labels.keys())
```

Comment:
```suggestion
        labels = sorted(layer.predefined_labels)
```

Maybe we should store predefined labels in sorted order?